src/themes/default/elements/icon.overrides

```diff
@@ -201,6 +202,19 @@ i.icon.other.gender.horizontal:before { content: "\f22b"; }
 i.icon.neuter:before { content: "\f22c"; }
 i.icon.genderless:before { content: "\f22d"; }
 
+/* Accessibility */
```

Comment:
New section: Accessibility. With a bunch of new icons that depict accessible technologies.